app/models/message_draft.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Message drafts are essentially messages with an implied folder (drafts)
+class MessageDraft < Message
+end
```

Comment:
Well, what I mean to say is, Message is going to have more validations than draft does. A draft might need subject "yet" whereas MessageDraft will need subject before it is valid. So if you inherit from Message, you will inherit more validations than you need. But we can address that stuff in a separate PR.